test/integration-tests/dev_double_trailing_slash.exp

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/expect -f
+
+log_user 0
+set force_conservative 1  ;# set to 1 to force conservative mode even if
+;# script wasn't run conservatively originally
+if {$force_conservative} {
+    set send_slow {1 .1}
+    proc send {ignore arg} {
+        sleep .3
+        exp_send -s -- $arg
+    }
+}
+
+set timeout -1
+spawn node ./build/index.js viz
+match_max 100000
+send -- "my-viz\r"
```

Comment:
Are these two sends in response to two different prompts from the `dscc-gen` command? If so, you should insert an `expect` between them, to await the prompt for the bucket name.